assets/xml/objects/object_ob.xml

```diff
@@ -1,84 +1,100 @@
+        <Texture Name="gMoonChildTshirtDetailTex" OutName="moonchild_tshirt_detail_tex" Format="ci8" Width="16" Height="32" Offset="0x15E68" />
+        <Texture Name="gMoonChildHairTex" OutName="moonchild_hair" Format="ci8" Width="8" Height="16" Offset="0x16068" />
+        <Texture Name="gMoonChildEarTex" OutName="moonchild_ear" Format="ci8" Width="8" Height="8" Offset="0x160E8" />
+        <Texture Name="gMoonChildBackHair" OutName="moonchild_back_hair" Format="ci8" Width="8" Height="16" Offset="0x16128" />
+        <Texture Name="gMoonChildTshirtNecklineTex" OutName="moonchild_tshirt_neckline" Format="ci8" Width="16" Height="32" Offset="0x161A8" />
+
+        <!-- Moonchild Limbs -->
+        <Limb Name="gMoonChildRootLimb" Type="Standard" EnumName="MOONCHILD_LIMB_ROOT" Offset="0x163A8" />
+        <Limb Name="gMoonChildBottomChestLimb" Type="Standard" EnumName="MOONCHILD_LIMB_BOTTOM_CHEST" Offset="0x163B4" />
+        <Limb Name="gMoonChildUpperChestLimb" Type="Standard" EnumName="MOONCHILD_LIMB_UPPER_CHEST" Offset="0x163C0" />
+        <Limb Name="gMoonChildLeftArmLimb" Type="Standard" EnumName="MOONCHILD_LIMB_LEFT_ARM" Offset="0x163CC" />
+        <Limb Name="gMoonChildLeftForearmLimb" Type="Standard" EnumName="MOONCHILD_LIMB_LEFT_FOREARM" Offset="0x163D8" />
+        <Limb Name="gMoonChildLeftHandLimb" Type="Standard" EnumName="MOONCHILD_LIMB_LEFT_HAND" Offset="0x163E4" />
+        <Limb Name="gMoonChildRightArmLimb" Type="Standard" EnumName="MOONCHILD_LIMB_RIGHT_ARM" Offset="0x163F0" />
+        <Limb Name="gMoonChildRightForearmLimb" Type="Standard" EnumName="MOONCHILD_LIMB_RIGHT_FOREARM" Offset="0x163FC" />
+        <Limb Name="gMoonChildRightHandLimb" Type="Standard" EnumName="MOONCHILD_LIMB_RIGHT_HAND" Offset="0x16408" />
+        <Limb Name="gMoonChildHeadLimb" Type="Standard" EnumName="MOONCHILD_LIMB_HEAD" Offset="0x16414" />
+        <Limb Name="gMoonChildPelvisLimb" Type="Standard" EnumName="MOONCHILD_LIMB_PELVIS" Offset="0x16420" />
+        <Limb Name="gMoonChildLeftThighLimb" Type="Standard" EnumName="MOONCHILD_LIMB_LEFT_THIGH" Offset="0x1642C" />
+        <Limb Name="gMoonChildLeftLegLimb" Type="Standard" EnumName="MOONCHILD_LIMB_LEFT_LEG" Offset="0x16438" />
+        <Limb Name="gMoonChildLeftFootLimb" Type="Standard" EnumName="MOONCHILD_LIMB_LEFT_FOOT" Offset="0x16444" />
+        <Limb Name="gMoonChildRightThighLimb" Type="Standard" EnumName="MOONCHILD_LIMB_RIGHT_THIGH" Offset="0x16450" />
+        <Limb Name="gMoonChildRightLegLimb" Type="Standard" EnumName="MOONCHILD_LIMB_RIGHT_LEG" Offset="0x1645C" />
+        <Limb Name="gMoonChildRightFootLimb" Type="Standard" EnumName="MOONCHILD_LIMB_RIGHT_FOOT" Offset="0x16468" />
+
+        <!-- Moonchild Skeleton -->
+        <Skeleton Name="gMoonChildSkel" Type="Flex" LimbType="Standard" LimbNone="MOONCHILD_LIMB_NONE" LimbMax="MOONCHILD_OB_LIMB_MAX" EnumName="object_ob_Limbs" Offset="0x164B8" />
```

Comment:
I'm guessing you didn't mean to include that `OB` in the `LIMB_MAX` name, and you can also make an EnumName for this:
```suggestion
        <Skeleton Name="gMoonChildSkel" Type="Flex" LimbType="Standard" LimbNone="MOONCHILD_LIMB_NONE" LimbMax="MOONCHILD_LIMB_MAX" EnumName="MoonchildLimb" Offset="0x164B8" />
```